UnityProject/Assets/Scripts/Weapons/Melee/StunBaton.cs

```diff
@@ -0,0 +1,61 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.Networking;
+
+[RequireComponent(typeof(Pickupable))]
+[RequireComponent(typeof(StunBatonActivate))]
+public class StunBaton : NetworkBehaviour
+{
+	public SpriteRenderer spriteRenderer;
+
+	/// <summary>
+	/// Sound played when turning this baton on/off
+	/// </summary>
+	public string soundToggle;
```

Comment:
All these don't need to be public
```suggestion
	[SerializeField]
	private string soundToggle;
```